server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/DelegateGetMethodContract.scala

```diff
@@ -404,23 +403,16 @@ trait DelegateGetMethodContract {
       .isEqualTo(
       s"""{
          |  "sessionState": "${SESSION_STATE.value}",
-         |  "methodResponses": [[
-         |    "Delegate/get",
-         |    {
-         |      "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
-         |      "list": [
-         |        {
-         |          "id": "$delegateId1",
-         |          "username": "bob@domain.tld"
-         |        },
-         |        {
-         |          "id": "$delegateId2",
-         |          "username": "cedric@domain.tld"
-         |        }
-         |      ],
-         |      "notFound": []
-         |    },
-         |    "c1"]]
+         |  "methodResponses": [
+         |        [
+         |            "error",
+         |            {
+         |                "type": "forbidden",
+         |                "description": "Access to other accounts is forbidden"
+         |            },
+         |            "c1"
+         |        ]
+         |    ]
```

Comment:
Can you help align `bobCanOnlyManageHisPrimaryAccountSetting` (DelegateSetContract) to this response?